### PR TITLE
KAA-1578: File system log appender - reviewed.

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Programming-guide/Key-platform-features/Data-collection/File-system-log-appender/index.md [hunk 1]
```diff
@@ -11,69 +10,38 @@ sort_idx: 40
...
+	
+	![File system log appender](attach/file-system-log-appender.png)
 
-It is also possible to create a file system log appender for your application by using [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Logging/editLogAppender).
-The following example illustrates how to create the file system log appender via Admin REST API.
+4. Fill in the **Configuration** section for your log appender and click **Add**.
+See [Configure log appender](#configure-log-appender).
 
-## Configuration
+Alternatively, you can use the [server REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Logging/editLogAppender) to create or edit your Couchbase log appender.
```
Comment (Polinaa) [inline doc/Programming-guide/Key-platform-features/Data-collection/File-system-log-appender/index.md]: File system

File: doc/Programming-guide/Key-platform-features/Data-collection/File-system-log-appender/index.md [hunk 2]
```diff
@@ -125,49 +93,66 @@ Example result:
...
+    "encoderPattern":"%-4relative [%thread] %-5level %logger{35} - %msg%n"
+}
+```
+
+
+## Playing with file system log appender
+
+The example below uses the **Data collection demo** from [Kaa Sandbox]({{root_url}}Glossary/#kaa-sandbox).
+
+To play around with the the Couchbase log appender:
```
Comment (Polinaa) [inline doc/Programming-guide/Key-platform-features/Data-collection/File-system-log-appender/index.md]: File system log appender

**Review by Polinaa — CHANGES_REQUESTED** [review]

File: doc/Programming-guide/Key-platform-features/Data-collection/File-system-log-appender/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Programming-guide/Key-platform-features/Data-collection/File-system-log-appender/index.md]: Fixed.

File: doc/Programming-guide/Key-platform-features/Data-collection/File-system-log-appender/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Programming-guide/Key-platform-features/Data-collection/File-system-log-appender/index.md]: Fixed.

**Review by hunch7 — COMMENTED** [review]
Fixed upon QA review.

**Review by Polinaa — APPROVED** [review]

**Comment by vadimol** [comment]
Jenkins and Travis fails are not related.

**Review by vadimol — APPROVED** [review]

**Pull request merged**